### DISCUSSION TITLE
How to access assets directly?

### DISCUSSION THREAD

**Discussion by jamiepickett** [datenstrom/community#619] [Ask a question] [answered]
I currently have a site that if backed by wordpress and I am in the process of putting together an MVP for a replacement.  I have a need to access content assets directly, however it seems that is being prevented.  Is there a way that I can reference for example: http://localhost:8888/articles/3d-printing-for-iot-developers/images/Figure1.jpg without Yellow returning a 404 page to me?

On a separate note, the site I am looking to replace is somewhat high profile and I have a few other functions that I am looking to be able to achieve.  Is there someone from the core team that I can email or have email me to discuss capabilities?

Thanks,
J

**Comment by schulle4u** [comment, marked as answer]
Yellow organizes content and assets in different folders, I think migrating from another structure will be a little more difficult. See also [this help page](https://datenstrom.se/yellow/help/adjusting-media) on how to adjust media. 

Commercial support is [available here](https://mayberg.se/support/).
